doc/introduction/index.rst

```diff
@@ -408,6 +411,15 @@ The Boot process proceeds as follows:
    services shall not use IOMMU. In addition, currently we only support
    UEFI timer with HPET MSI.
 
+In this boot mode, both the Service and User VM boot options (e.g. Linux
+command-line parameters) are configured following the instructions for the EFI
+bootloader used by the Operating System (OS).
+
```

Comment:
Do we need to care about VSBL support for UOS? The boot option is defined in UOS launch script in that case.

Reply:
Suggest we only mention Service VM boot here.

Reply:
I think we need to cover both (eventually). But there is no mention at all about (v)SBL in this document at this stage.

Does booting the Service VM from SBL follow the same sequence as described in the [De-privilege boot mode](https://projectacrn.github.io/latest/introduction/index.html#de-privilege-boot-mode) section?

Reply:
Part of the platform specific UOS cmdline (such as acrn-gt) from the launch script when the guest boot loader is vSBL.
But for SBL to boot SOS, the cmdline is stitched to the boot image.